internal/lua_tests/lua_api_test.go

```diff
@@ -14,8 +14,8 @@ import (
 
 type luaTest struct {
 	file       string
-	beforeTest func() error
-	afterTest  func() error
+	beforeTest func(t *testing.T) error
+	afterTest  func(t *testing.T, revertNum int) error
```

Comment:
Note that this structure has been designed to suit well for any kind of tests (think of passing `interface` or unlimited number of arguments)